lib/p5.play.js

```diff
@@ -1241,6 +1241,36 @@ function Sprite(pInst, _x, _y, _w, _h) {
   */
   this.animation = undefined;
 
+  /**
+   * Swept collider oriented along the current velocity vector, extending to
+   * cover the old and new positions of the sprite.
+   *
+   * The corners of the swept collider will extend beyond the actual swept
+   * shape, but it should be sufficient for broad-phase detection of collision
+   * candidates.
+   *
+   * Note that this collider will have no dimensions if the source sprite has no
+   * velocity.
+   */
+  this._sweptCollider = new p5.OrientedBoundingBoxCollider();
+
+  /**
+   * If the sprite is moving, use the swept collider. Otherwise use the actual
+   * collider.
+   */
+  this._getBroadPhaseCollider = function() {
+    return (this.velocity.magSq() > 0) ? this._sweptCollider : this.collider;
+  };
```

Comment:
If this is just a broadphase check, does it matter if the broadphase collider changes? Shouldn't the real collider just report a non-collision if the broadphase bounds were too big?